plugins/modules/route53_info.py

```diff
@@ -376,7 +376,7 @@
     version_added: 4.1.0
     version_added_collection: community.aws
 health_check:
-    description: A dict of Route53 health check details returned by get_health_check_status in boto3.
+    description: A dict of Route53 health check details returned by get_health_check in boto3.
```

Comment:
```suggestion
    description: A dict of Route53 health check details returned by get_health_check_status in boto3.
```

The boto3 call is get_health_check_status (API call is GetHealthCheckStatus)

Reply:
@tremble thank you for reviewing the PR.

this is correction in the documentation I'd like to make because
[GetHealthCheckStatus](https://docs.aws.amazon.com/Route53/latest/APIReference/API_GetHealthCheckStatus.html) returns [HealthCheckObservations](https://docs.aws.amazon.com/Route53/latest/APIReference/API_GetHealthCheckStatus.html#Route53-GetHealthCheckStatus-response-HealthCheckObservations) while [GetHealthCheck](https://docs.aws.amazon.com/Route53/latest/APIReference/API_GetHealthCheck.html) returns [HealthCheck](https://docs.aws.amazon.com/Route53/latest/APIReference/API_GetHealthCheck.html#Route53-GetHealthCheck-response-HealthCheck)